### ISSUE TITLE
comments

### ISSUE THREAD

**Issue by ghost** [UBC-MDS/525_group11#17]
The notebook needs to be better organized: you could have kept the rubric from the wording.
It is hard to follow what's being done as there are very few comments on the code, and not enough detailed analysis of your observations/results.

**Comment by Saule-Atymtayeva** [comment]
Thank you for your feedback, Paul! @paul-TA

We've decided to put our observations under each task as it's easier to read the code and our conclusions in one place.
Regarding the comments, we tried to provide the comments as much as we could but not overload the code. So, in cases where there are no comments, we printed the results of the code.

We would appreciate it if you give us more detailed feedback so that we'll prevent such flaws in the future.
